### PR TITLE
Fixes #20738: Once we decided on which generic method we focus report in a block, it shows the uuid rather than its name

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/20738

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/ViewBlock.elm [hunk 1]
```diff
@@ -232,9 +232,10 @@ showBlockTab model parentId block uiInfo techniqueUi =
                                        Call _ c -> Maybe.withDefault (c.methodName.value) (Maybe.map .name (Dict.get c.methodName.value model.methods))
                                    else
                                      componentValue
+
                      in
                        element "li"
-                               |> addActionStopAndPrevent ("click", MethodCallModified (Block parentId {block | reportingLogic = FocusReport (getId child).value }))
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/ViewBlock.elm:237]: That should still be the id. The id needs to be used in the focus report, not the component name.

You should change the display

**Comment by ElaadF** [comment]
Commit modified

**Review by VinceMacBuche — APPROVED** [review]

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**